src/uniparc/components/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,291 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, ExternalLink, LongNumber } from 'franklin-sites';
+import cn from 'classnames';
+
+import HTMLHead from '../../../shared/components/HTMLHead';
+// import YouTubeEmbed from '../../../shared/components/YouTubeEmbed';
+
+import useDataApi from '../../../shared/hooks/useDataApi';
+
+import {
+  LocationToPath,
+  Location,
+  getLocationEntryPath,
+} from '../../../app/config/urls';
+import ftpUrls from '../../../shared/config/ftpUrls';
+import { stringifyQuery } from '../../../shared/utils/url';
+import apiUrls from '../../../shared/config/apiUrls/apiUrls';
+
+import { SearchResults } from '../../../shared/types/results';
+import { Namespace } from '../../../shared/types/namespaces';
+import { FacetsEnum } from '../../config/UniParcFacetConfiguration';
+
+import styles from './styles/landing-page.module.scss';
+
+import ArchiveIllustration from '../../../images/archive_illustration.img.svg';
+import {
+  useMediumScreen,
+  useSmallScreen,
+} from '../../../shared/hooks/useMatchMedia';
+
+// const tutorialsInfo = [
+//   {
+//     id: 'yp1O1gDK8oA',
+//     title: 'How to search UniProtKB',
+//     date: new Date('2021-10-26'),
+//   },
+//   {
+//     id: 'BHu88Sv--mc',
+//     title: 'How to explore a UniProt entry',
+//     date: new Date('2022-02-17'),
+//   },
+//   {
+//     id: 'p4_gGkM-Rfs',
+//     title: 'How to download embeddings in UniProt',
+//     date: new Date('2023-06-25'),
+//   },
+// ];
+
+const metaDescription =
+  'The UniProt sequence archive (UniParc) is a comprehensive and non-redundant database that contains most of the publicly available protein sequences in the world';
+
+const LandingPage = () => {
+  const [expanded, setExpanded] = useState(false);
+  const smallScreen = useSmallScreen();
+  const mediumScreen = useMediumScreen();
+  const { data } = useDataApi<SearchResults<never>>(
+    apiUrls.search.search({
+      namespace: Namespace.uniparc,
+      query: '*',
+      size: 0,
+      facets: [FacetsEnum.Database],
+    })
+  );
+
+  const databases = Array.from(data?.facets?.[0].values || []).sort(
+    (a, b) => b.count - a.count
+  );
+
+  const groupedDatabases = [];
+  if (!expanded) {
+    groupedDatabases.push(databases.slice(0, 5));
+  } else {
+    let nGroups = 3;
+    if (mediumScreen) {
+      nGroups = 2;
+    }
+    if (smallScreen) {
+      nGroups = 1;
+    }
+    const countByColumn = Math.ceil(databases.length / nGroups);
+    let start = 0;
+    while (start < databases.length) {
+      groupedDatabases.push(databases.slice(start, start + countByColumn));
+      start += countByColumn;
+    }
+  }
+
+  return (
+    <div className={styles['landing-page']}>
+      <HTMLHead title="UniProt sequence archive (UniParc)">
+        <meta name="description" content={metaDescription} />
+      </HTMLHead>
+      <section className="uniprot-grid">
```

Comment:
As we have a decent amount of whitespace on the page what do you think about increase the grid gap to 2rem just to make clearer the different sections of the page?

Reply:
For that I'm not too keen, because it's using the franklin-defined grid and it would mean the grid is inconsistent here compared to the rest of the website.
Hopefully we get more content in the future in order to fill this page